spec/system/pokemon_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe "Pokemon", type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  it "enables me to create pokemon" do
+    visit "/pokemon/new"
```

Comment:
Du kannst statt dem String den Pfad erzeugen lassen -> https://guides.rubyonrails.org/routing.html#path-and-url-helpers
```suggestion
    visit "new_pokemon_path"
```
Ein bisschen Context: Mit `rails routes` kannst du ja dir Infos zu deinen Routen ansehen, da schau mal nach den `prefix`. An diesen Prefix kannst du dann `_path` oder `_url` anhängen.